stage.sh

```diff
@@ -154,10 +154,11 @@ if [ "$only_cfn_template" = false ]; then
     fi
 
     # Sync product images
-    echo " + Copying product images"
-    aws s3 sync s3://retail-demo-store-code/datasets/1.3/images/ s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC || echo "Skipping load of remote image dataset 1.3"
-    aws s3 sync s3://retail-demo-store-code/datasets/1.4/images/ s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC || echo "Skipping load of remote image dataset 1.4"
-    aws s3 sync datasets/1.4/images/ s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC || echo "Skipping load of local image dataset 1.4"
+    #echo " + Copying product images"
+    # disabled, images are downloaded from https://code.retaildemostore.retail.aws.dev/images.tar.gz
+    #aws s3 sync s3://retail-demo-store-code/datasets/1.3/images/ s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC || echo "Skipping load of remote image dataset 1.3"
+    #aws s3 sync s3://retail-demo-store-code/datasets/1.4/images/ s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC || echo "Skipping load of remote image dataset 1.4"
+    #aws s3 sync datasets/1.4/images/ s3://${BUCKET}/${S3PATH}images/ $S3PUBLIC || echo "Skipping load of local image dataset 1.4"
```

Comment:
Why aren't these lines being deleted? Do we need a new (separate) script that updates the zip when images change?

Reply:
I was in the initial idea that we would have the public buckets back at some point. but yes the s3 sync should be deleted.
The images are always pulled from the images.tar.gz.

But, I should have kept the line 160 as I believe this for custom images?

In case you want to customise the base images, you would have to create your own images.tar.gz and reference it in cloudformation.

The other way was to keep the existing behaviour and make a branch if we have a .tar.gz, I tried to keep things simple